From 1fe6f0def67fc76344ecd7c4e4b038b8d7cc7197 Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Thu, 23 Oct 2025 19:35:59 +0200
Subject: [PATCH 18/19] ipa/simple: Add tuning file for IMX858

Used by the Fairphone 5. The value is guessed from similar IMX sensors.
---
 src/ipa/simple/data/imx858.yaml | 11 +++++++++++
 src/ipa/simple/data/meson.build |  1 +
 2 files changed, 12 insertions(+)
 create mode 100644 src/ipa/simple/data/imx858.yaml

diff --git a/src/ipa/simple/data/imx858.yaml b/src/ipa/simple/data/imx858.yaml
new file mode 100644
index 000000000..f7d01b738
--- /dev/null
+++ b/src/ipa/simple/data/imx858.yaml
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: CC0-1.0
+%YAML 1.1
+---
+version: 1
+algorithms:
+  - BlackLevel:
+      blackLevel: 4096
+  - Awb:
+  - Lut:
+  - Agc:
+...
diff --git a/src/ipa/simple/data/meson.build b/src/ipa/simple/data/meson.build
index e0c8e443a..8536a9672 100644
--- a/src/ipa/simple/data/meson.build
+++ b/src/ipa/simple/data/meson.build
@@ -6,6 +6,7 @@ conf_files = files([
     'imx363.yaml',
     'imx371.yaml',
     'imx376.yaml',
+    'imx858.yaml',
     's5k3l6xx.yaml',
     'uncalibrated.yaml',
 ])
-- 
2.51.0

